hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -131,90 +131,90 @@ struct Vunit;
   O(nop, Inone, Un, Dn)\
   O(ud2, Inone, Un, Dn)\
   /* arithmetic instructions */\
```

Comment:
I considered many different ways to annotate a Vinstr with information about the Vinstrs that use the VregSF that it defs. Storing it in the Vunit was problematic if the blocks were mutated at all, changing their index in the block. I also used the pointers to the Vinstrs, but those also change often with copy constructors. Storing it in the Vinstr itself made the most sense. Vinstrs have restrictions on their size, so I opted to put required flags only in the Vinstrs that def a VregSF. Furthermore, I used a uint8_t since no platforms should have more than 8 status flags.